README.md

```diff
@@ -305,6 +306,30 @@ method takes one argument:
 
 2. `chain` must return a value of the same Chain
 
+### ChainRec
+
+A value that implements the ChainRec specification must also implement the Chain specification.
+
+1. `T.chainRec((done, next, value) => g(done(value)), i) `is equivalent to `g(i)`
```

Comment:
`g` takes any type it's like `T.of` but I did not used `T.of` as `Chain` does not requite the method `of`.

i.e. this are equivalent:
`Identity.chainRec((done, next, value) => Identity.of(done(value)), 1)` ≡ `Identity.of(1)`